app/src/main/java/com/amaze/filemanager/ui/views/drawer/Drawer.java

```diff
@@ -1,3 +1,25 @@
+/*
+ * Drawer.java
+ *
+ * Copyright (C) 2014-2018 Mitch West<mitch9378dev@gmail.com> and Contributors.
```

Comment:
> I am not sure if I handled this license correctly. Please let me know if I need to change anything.

Recommended to put other developer's name before yours.

For an example, see this:

https://github.com/TeamAmaze/AmazeFileManager/blob/b3197cf7f10a50574b7ddc478f7713fec996841f/app/src/main/java/com/amaze/filemanager/activities/MainActivity.java#L4-L5

Reply:
Thanks, I wasn't really sure who to put there but I will fix it.